Unable to build Lemmur using flutter

### Describe the bug

Unable to build .apk with flutter

### Steps to reproduce

1. Git clone the repo
2. cd into the folder
3. execute the provided buil command
4. See error

### Relevant logs

Target file "lib/main.dart" not found.>

### Expected behavior

Getting built version of the app

![ksnip_20220106-173127](https://user-images.githubusercontent.com/31543930/148407564-d0f55bb8-24e1-4d18-91d3-aafaca6ad9c6.png)

Now I have the problem 

`FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:packageProdRelease'.
> A failure occurred while executing com.android.build.gradle.tasks.PackageAndroidArtifact$IncrementalSplitterRunnable
   > SigningConfig "prod" is missing required property "storeFile".
`

--- a/README.md
+++ b/README.md
@@ -36,9 +36,9 @@ A mobile client for [Lemmy](https://github.com/LemmyNet/lemmy) - a federated red
 
 ### Android
 
-1. Build: `flutter build apk`
+1. Build: `flutter build apk --flavor prod --target lib/main_prod.dart --release`
 
-The apk will be in `build/app/outputs/flutter-apk/app-release.apk`
+The apk will be in `build/app/outputs/flutter-apk/app-prod-release.apk`
 
 ### Linux
 
@@ -47,9 +47,9 @@ The apk will be in `build/app/outputs/flutter-apk/app-release.apk`
    ```sh
    flutter config --enable-linux-desktop
    ```
-3. Build: `flutter build linux`
+3. Build: `flutter build linux --target lib/main_prod.dart --release`
 
-The executable will be in `build/linux/release/bundle/lemmur` (be aware, however, that this executable is not standalone)
+The executable will be in `build/linux/x64/release/bundle/lemmur` (be aware, however, that this executable is not standalone)
 
 ### Windows
 
@@ -58,7 +58,7 @@ The executable will be in `build/linux/release/bundle/lemmur` (be aware, however
    ```sh
    flutter config --enable-windows-desktop
    ```
-3. Build: `flutter build windows`
+3. Build: `flutter build windows --target lib/main_prod.dart --release`
 
 The executable will be in `build\windows\runner\Release\lemmur.exe` (be aware, however, that this executable is not standalone)
 

--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -54,8 +54,6 @@ android {
     }
 
     signingConfigs {
-        dev { }
-
         if (System.getenv("ANDROID_STORE_PATH")) {
             prod {
                 keyAlias System.getenv("ANDROID_KEY_ALIAS")
@@ -64,7 +62,12 @@ android {
                 storePassword System.getenv("ANDROID_STORE_PASSWORD")
             }
         } else {
-            prod { }
+            prod {
+                keyAlias signingConfigs.debug.keyAlias
+                keyPassword signingConfigs.debug.keyPassword
+                storeFile signingConfigs.debug.storeFile
+                storePassword signingConfigs.debug.storePassword
+            }
         }
     }
 

